src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -18,6 +18,10 @@
     public static String getResource(String key) {
         return TestResource.getBundle(Constants.MSSQL_JDBC_PACKAGE + ".TestResource").getString(key);
     }
+    
+    public static String formatErrorMsg(String resource) {
+        return (".*\\Q" + getResource(resource) + "\\E").replaceAll("\\{+[0-9]+\\}", "\\\\E.*\\\\Q");
+    }
```

Comment:
in TestUtils

Reply:
Different bundle, the testutils one gets it from SQLServerResource.